airbyte-integrations/connectors/source-fleetio/README.md

```diff
@@ -0,0 +1,85 @@
+# Fleetio Source
+
+This is the repository for the Fleetio configuration based source connector.
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.com/integrations/sources/fleetio).
+
+## Fleetio Overview
+View the Fleetio website [here](https://fleetio.com). For for information the Fleetio API, check out out [Fleetio API Portal](https://developer.fleetio.com).
+
+## Local development
+
+#### Building via Gradle
```

Comment:
@alafanechere, whoa whoa whoa, I think we don't use Gradle for connectors anymore, right? Should we replace this with `airbyte-ci` instructions? Can you copy over the snippet in here in a suggestion, please?

Reply:
[I suggest following a structure like we have on `source-stripe` 
](https://github.com/airbytehq/airbyte/blob/main/airbyte-integrations/connectors/source-stripe/README.md#L37)